model/task/db.go

```diff
@@ -95,6 +95,7 @@ var (
 	GenerateTasksErrorKey       = bsonutil.MustHaveTag(Task{}, "GenerateTasksError")
 	GeneratedTasksToActivateKey = bsonutil.MustHaveTag(Task{}, "GeneratedTasksToActivate")
 	ResetWhenFinishedKey        = bsonutil.MustHaveTag(Task{}, "ResetWhenFinished")
+	RestartFailedKey            = bsonutil.MustHaveTag(Task{}, "RestartFailed")
```

Comment:
For consistency would it make sense to call this ResetFailedWhenFinished so it better matches ResetWhenFinished?